test/infamy/netconf.py

```diff
@@ -435,5 +419,12 @@ def get_current_time_with_offset(self):
         unadjusted time.
         """
         data = self.get_data("/ietf-system:system-state/clock", parse=False)
+        parsed_data = lxml.etree.fromstring(data)
+
         xpath = './/{urn:ietf:params:xml:ns:yang:ietf-system}current-datetime'
-        return data.data_ele.find(xpath).text
+        current_datetime = parsed_data.find(xpath)
+
+        if current_datetime is not None:
+            return current_datetime.text
+        else:
+            raise ValueError("current-datetime element not found in the response")
```

Comment:
It's a good practice to ensure all files have a final newline.  (It's been removed in this patch.)